sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyDetection/MetricSeriesGroupDetectionCondition.cs

```diff
@@ -16,16 +17,20 @@ public partial class MetricSeriesGroupDetectionCondition : MetricWholeSeriesDete
         /// <summary>
         /// Initializes a new instance of the <see cref="MetricSeriesGroupDetectionCondition"/> class.
         /// </summary>
-        public MetricSeriesGroupDetectionCondition()
+        /// <param name="seriesGroupKey">The key that identifies the group of time series to which these conditions apply within a metric. A subset of the possible dimensions of the associated data feed must be set.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="seriesGroupKey"/> is <c>null</c>.</exception>
+        public MetricSeriesGroupDetectionCondition(DimensionKey seriesGroupKey)
```

Comment:
nit: do you have tests to check that we correctly assert when null? 
Same for MetricSingleSeriesDetectionCondition

Reply:
We don't, but I think it's common to not add constructor assertion tests to every single model, right? We don't do it in FR either. Let me know if you think otherwise.